### PR TITLE
Update affiliations

### PR DESCRIPTION
**Pull request by jasongrout**
## Questions to answer ❓

**Background or context to help others understand the change.**

Some affiliations have changed for steering council members. This PR updates them

**A brief summary of the change.**

Update the affiliations

**What is the reason for this change?**

**Alternatives to making this change and other considerations.**

## The process ❗

The process for changing the governance pages is as follows:

* Open a pull request **in draft state**. This triggers a discussion and iteration phase
  for your proposed changes.
* When you believe enough discussion has happened,
  **move the pull request to an active state**. This triggers a vote.
* During the voting phase, no substantive changes may be made to the pull request.
* The Steering Council will vote, and at the end of voting the pull request is merged or closed.

The discussion phase is meant to gather input and multiple perspectives from the community.
Make sure that the community has had an opportunity to weigh in on
the change **before calling a vote**. A good rule of thumb is to ask several steering council
members if they believe that it is time for a vote, and to let at least one person review
the pull request for structural quality and typos.

### CODE REVIEW

**Comment by jasongrout** [comment]
It looks like others affiliations may also be out of date. I didn't want to take the liberty of updating affiliations without approval, so if you comment here with your current affiliation, I'm happy to make the change.

**Comment by blink1073** [comment]
Thanks @jasongrout, I updated my affiliation as well.

**Comment by blink1073** [comment]
Looks like my change was overwritten, my entry should say MongoDB.

**Comment by jasongrout** [comment]
> Looks like my change was overwritten, my entry should say MongoDB.

Oops, sorry, I didn't see you had made a second commit and thought I had made a mistake and I force-pushed my change back. I'll revert my force-push.

**Comment by jasongrout** [comment]
I opened a similar PR to the website: https://github.com/jupyter/jupyter.github.io/pull/442

**Comment by ivanov** [comment]
Thanks @jasongrout - I updated the Noteable folks, alphabetized everything, moved the institutional partner without remaining steering council members to Tier 2, and removed a previous institutional partner that last had contributors removed in 2015 (but remained listed per [grace period of 1 year in governance.md](https://github.com/jupyter/governance/blob/master/governance.md?plain=1#L283-L285))

File: people.md [hunk 1]
```diff
@@ -62,20 +62,23 @@ Alphabetical by first name:
...
-- [Bloomberg](https://www.bloomberg.com/) (Jason Grout, Paul Ivanov)
+
+- [Amazon Web Services](https://aws.amazon.com/) (Brian Granger)
 - [Cal Poly](https://www.calpoly.edu/) (Ana Ruvalcaba)
-- [Netflix](https://www.netflix.com/) (Kyle Kelley, M Pacer)
-- [QuantStack](http://quantstack.net/) (Sylvain Corlay)
+- [Databricks](https://databricks.com/) (Jason Grout)
+- [MongoDB](https://www.mongodb.com/) (Steven Silvester)
+- [Netflix](https://www.netflix.com/) (M Pacer)
+- [Noteable](https://noteable.io/) (Carol Willing, Paul Ivanov, Kyle Kelley)
```
Comment (rgbkrk) [inline people.md:71]: Thanks!

**Review by rgbkrk — APPROVED** [review]

File: people.md [hunk 2]
```diff
@@ -62,20 +62,23 @@ Alphabetical by first name:
...
 - [QuanSight](https://www.quansight.com/) (Matthias Bussonnier, Carol Willing)
+- [QuantStack](http://quantstack.net/) (Sylvain Corlay)
```
Comment (damianavila) [inline people.md:73]:
```suggestion
- [QuantStack](http://quantstack.net/) (Sylvain Corlay)
- [The International Interactive Computing Collaboration](https://2i2c.org/) (Damian Avila)
```

Comment (damianavila) [reply people.md:73]: Updated my affiliation as well :wink:

**Comment by jasongrout** [comment]
Merged. Thanks everyone!

**Pull request merged**